### PR TITLE
ELU added to Ivy Stateful Api

### PR DESCRIPTION
**Pull request by tejeshbhalla**
#19747

### CODE REVIEW

**Comment by tejeshbhalla** [comment]
@vedpatwardhan  in the sub_backends elaboration it was failing due to it opening .ds_store on my macbook as it is not a directory and that was the fix i made

File: ivy/utils/backend/sub_backend_handler.py [hunk 1]
```diff
@@ -22,6 +22,8 @@
 ):
     if backend.startswith("__"):
         continue
+    if backend.endswith(".DS_Store"):
+        continue
```
Comment (CatB1t) [inline ivy/utils/backend/sub_backend_handler.py:26]: We don't need new line as a check, also it's better to create a seperate issue and PR to get this fixed, @Infrared1029 can provide more insight on a better solution than hardcoding `DS_STORE` into there.

**Review by CatB1t — CHANGES_REQUESTED** [review]

File: ivy/utils/backend/sub_backend_handler.py (hunk 1 above)
Comment (tejeshbhalla) [reply ivy/utils/backend/sub_backend_handler.py:26]: Sure

**Review by vedpatwardhan — COMMENTED** [review]
I completely agree with what @CatB1t said, let's see what @Infrared1029 thinks. Thanks @tejeshbhalla 😄

**Comment by tejeshbhalla** [comment]
@Mr-Niraj-Kulkarni could you explain whats wrong with the tests ?

**Comment by Mr-Niraj-Kulkarni** [comment]
> @Mr-Niraj-Kulkarni could you explain what's wrong with the tests?
Everything looks good to me according to the code.

**Review by Mr-Niraj-Kulkarni — APPROVED** [review]

**Pull request merged**